extensions/wikia/Recirculation/services/CurationCMSService.class.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+class CurationCMSService {
```

Comment:
IMO we should create this class in the same place like for example ArticleVideoService class, `includes/wikia/services` or something like that

Reply:
and this controller should only use that service

Reply:
Let's do it after AB test.